jvm-observ-lib/main.libsonnet

```diff
@@ -15,16 +15,18 @@ local processlib = import 'process-observ-lib/main.libsonnet';
           filteringSelector: this.config.filteringSelector,
           groupLabels: this.config.groupLabels,
           instanceLabels: this.config.instanceLabels,
-          uid: this.config.uid,
+          uid: this.config.uid - '-jvm',
```

Comment:
A little hesitant on the UID change in case people blindly install it, would cause breaking changes to any links/bookmarks

Reply:
that is to make sure that process-lib metrics(and alerts), when attached more than once do not clash. 
In case of kafka lib:

jvm is imported two times. One for kafka itselft, second - for zookeeper(which is also java papp). that's why it is better to have uid dynamic.

Reply:
Ahhh that makes sense, thanks for explaining!